tox.ini

```diff
@@ -1,14 +1,14 @@
 [tox]
-envlist = py{27,34}-dj{16,17,18,19}
+downloadcache = .tox/_download/
+envlist =
+    py27-dj{19}
+    py35-dj{19}
 
 [testenv]
+passenv = CI CIRCLECI CIRCLE_*
+install_command=pip install -U -I {opts} {packages}
 commands = ./setup.py test
 basepython =
     py27: python2.7
-    py34: python3.4
-deps =
-    six
-    dj16: django==1.6.11
-    dj17: django==1.7.10
-    dj18: django==1.8.4
-    dj19: django==1.9.1
+    py35: python3.5
+deps = -r{toxinidir}/requirements.txt
```

Comment:
```
deps =
        dj19: Django>=1.9<1.11
```

Reply:
if I do this, I get many import errors when running tox..

Reply:
**reqiurements.txt**
```
six
```

**tox.init**

```
deps =
    -rrequirements.txt
    dj19: Django>=1.9<1.11
```

Reply:
also you need to fix `test_proj` and `compat` file as it is deprecated